test/core/rebalancingSetToken.spec.ts

```diff
@@ -335,6 +377,85 @@ contract('RebalancingSetToken', accounts => {
         await expectRevertError(subject());
       });
     });
+  });
+
+  describe('#mint called from Core', async () => {
+    let rebalancingToken: RebalancingSetTokenContract;
```

Comment:
As we discussed - a lot of this setup could be moved into a wrapper.

Reply:
Yep another PR methinks

Reply:
Yes and no, a lot of this stuff is unnecessary such as declaring a new `rebalancingToken` and `setName, setSymbol` (defined in coreWrapper already) that I noticed. I cleaned this up in my PR that should be on master. Makes following along with specs easier